packages/terra-demographics-banner/docs/themeable-variables.md

```diff
@@ -0,0 +1,8 @@
+# Themeable Variables
+
+Variables to `theme` the terra-demographics-banner component.
+
+| Variable Name                       | Default value   | Usage                                                        |
+|-------------------------------------|-----------------|--------------------------------------------------------------|
+| $demographics-background            | $terra-blue-100 | Background of the Demographics Banner.                       |
+| $demographics-background-deceased   | #000            | Background of the Demographics Banner for a deceased person. |
```

Comment:
Made the banner more themeable. https://github.com/cerner/terra-core/pull/204/commits/fd7fa0c471b7c8b82e16b084f456a4c3a0f99f0d